setup.py

```diff
@@ -144,6 +144,8 @@ def package_files(prefix: str, directory: str, extension: str):
     'wandb>=0.13.2,<0.14',
 ]
 
+extra_deps['comet_ml'] = ['comet_ml>=3.31.12']
```

Comment:
```suggestion
extra_deps['comet_ml'] = ['comet_ml>=3.31.12,<4.0.0']
```
Dependabot will auto-bump, and we want unit tests on major changes.